src/main/java/spoon/support/visitor/java/JavaReflectionVisitorImpl.java

```diff
@@ -46,38 +45,70 @@ public void visitPackage(Package aPackage) {
 		if (clazz.getPackage() != null) {
 			clazz.getPackage();
 		}
-		for (TypeVariable<Class<T>> generic : clazz.getTypeParameters()) {
-			visitTypeParameter(generic);
+		try {
+			for (TypeVariable<Class<T>> generic : clazz.getTypeParameters()) {
+				visitTypeParameter(generic);
+			}
+		} catch (Throwable ignore) {
+			// partial classpath
```

Comment:
can we catch more exact exception? Such error handling may cause big problems during future debugging.